docs/tailor/developer-guide/configuration.md

```diff
@@ -330,16 +330,10 @@ Add image
 
 ``` json
 meta: [{
```

Comment:
Could you please fix this part in all of the other `meta` ones as well (that are not part of this PR)? It should be wrapped in double-quotes.

Reply:
Done.